packages/core/src/Stores/gtm-store.js

```diff
@@ -33,6 +38,23 @@ export default class GTMStore extends BaseStore {
      */
     @computed
     get common_variables() {
+        const platform = () => {
+            const url_parsed = window.location.href.toLowerCase().split('&')[0].replace(/:\/\//,'/').split('/');
```

Comment:
You can use the native `URL` class, e.g.

- https://developer.mozilla.org/en-US/docs/Web/API/URL

```JavaScript
const url    = new URL(window.location.href);
const domain = url.hostname;
const path   = url.pathname;
```

Reply:
done